### ISSUE TITLE
Doesn't work twice... kills the process...

### ISSUE THREAD

**Issue by onzag** [tcql/node-yesno#7]
```
yesno = require("yesno");

(async () => {
        await yesno.askAsync("test");
        await yesno.askAsync("test");
        console.log("done")
})()
```

I expect the question to be asked twice.
The actual result is the program dies.
Node v11.15.0

**Comment by dosentmatter** [comment]
Happens to me too. For both sync and async `ask` functions on v10.15.3

**Comment by mreinstein** [comment]
Yeah, this definitely needs a refactor to support more than one prompt per invocation. PRs welcome!

**Comment by mreinstein** [comment]
published @ 0.2.1

**Comment by dosentmatter** [comment]
Thank you for fixing it so quickly!
